docs/3.0/common/reloadable.md

```diff
@@ -0,0 +1,31 @@
+:::option Reloadable
+
+<VersionReq version="3.3.6" />
+
+The reloadable option adds a reload icon next to the association title so users can easily reload just that turbo-frame instead of doing a full page reload.
+
+#### Usage
+To enable the `reloadable` feature, provide a block to the reloadable option when defining the field. Within this block, specify conditions under which the reloadable should be displayed.
```

Comment:
By default they can use a boolean. The block is for advanced usage when they want the behaviour to be more dynamic.

Reply:
OK, I will update it.